crates/router/src/connector/adyen/transformers.rs

```diff
@@ -417,7 +415,7 @@ impl TryFrom<types::PaymentsCancelResponseRouterData<AdyenCancelResponse>>
     fn try_from(
         item: types::PaymentsCancelResponseRouterData<AdyenCancelResponse>,
     ) -> Result<Self, Self::Error> {
-        let status = match item.response.response.as_str() {
+        let status = match item.response.status.as_str() {
             "received" => storage_enums::AttemptStatus::Voided,
             "processing" => storage_enums::AttemptStatus::Pending,
             _ => storage_enums::AttemptStatus::VoidFailed,
```

Comment:
This can go into a from implementation.
response_status -> adyen_status -> attempt_status